### DISCUSSION TITLE
flux2-sync /flux2 helm charts

### DISCUSSION THREAD

**Discussion by poblin-orange** [fluxcd/flux2#2177] [Q&A]
Hello,
just found out these charts
- https://artifacthub.io/packages/helm/fluxcd-community/flux2-sync
- https://artifacthub.io/packages/helm/fluxcd-community/flux2

Currently installing flux with kubectl apply from manifest. Id like to switch to helm
Any insight / documentation on the role of these 2 helm charts ?
Thks

**Comment by hiddeco** [comment]
This is an endorsed project maintained by the community, you can find more information about it here: https://github.com/fluxcd-community/helm-charts.

Given it is community maintained, support questions or other issues with the chart are best filled against the repository itself, as none of the Flux maintainers plays an active role in the maintenance of the chart.
